src/redux/store/index.js

```diff
@@ -7,6 +7,10 @@ const sagaMiddleware = createSagaMiddleware();
 
 const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
 
+if (window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__) {
+  window.BigInt.prototype.toJSON = function toJSON() { return this.toString(); };
```

Comment:
What and why is this?

Reply:
I'll add comments, but basically when you store BigInt in redux while also using devtools for redux the app will crash. This fixes that, and won't affect production software